mlflow/tracing/export/mlflow.py

```diff
@@ -37,48 +38,40 @@ def __init__(self, client: TraceClient):
         self._client = client
         self._trace_manager = InMemoryTraceManager.get_instance()
 
-    def export(self, spans: Sequence[MlflowSpanWrapper]):
+    def export(self, spans: Sequence[ReadableSpan]):
```

Comment:
note: The exporter now takes OTel's `ReadableSpan` not MLflow wrapper.

Reply:
Yay! This is awesome!